tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -615,10 +604,13 @@ class Converter {
   // Constructs a name and passed it to the TensorRT layer to support xprof.
   void SetLayerName(nvinfer1::ILayer* layer, const NodeDef& node_def,
                     absl::string_view sub_op_name = "",
-                    absl::optional<int> sub_op_instance = absl::nullopt);
+                    absl::optional<int> sub_op_instance = absl::nullopt,
+                    absl::optional<std::string> origin_node_name = absl::nullopt);
+
   void SetLayerName(nvinfer1::ILayer* layer, absl::string_view main_op_name,
                     absl::string_view sub_op_name,
-                    absl::optional<int> sub_op_instance = absl::nullopt);
+                    absl::optional<int> sub_op_instance = absl::nullopt,
+                    absl::optional<std::string> origin_node_name = absl::nullopt);
```

Comment:
I think this parameter is NOT added to the implementation, see the cc file. I couldn't comment in the implementation because it is not changed by this PR.

Reply:
Ah this was incorrectly applied to the second SetLayerName function instead of PrepareTensorForShape.  Fixed now, thanks